pulp_smash/constants.py

```diff
@@ -202,6 +202,11 @@
     http://docs.pulpproject.org/plugins/pulp_ostree/tech-reference/importer.html
 """
 
+OSTREE_FEED_SMALL = (
+    'https://repos.fedorapeople.org/pulp/pulp/fixtures/ostree/small/'
```

Comment:
OSTREE_FEED_SMALL  was removed, and OSTREE_FEED was updated to the URL related to data generated using pulp_fixtures.

Since these constants were used by 2 other tests, another PR was created to update these constants.